mockgcp/Makefile

```diff
@@ -144,7 +144,6 @@ gen-proto-no-fixup:
 		./third_party/googleapis/mockgrafeas/v1/*.proto \
 		./third_party/googleapis/mockgcp/cloud/bigquery/biglake/v1/*.proto \
 		./third_party/googleapis/mockgcp/cloud/bigquery/reservation/v1/*.proto \
-		./third_party/googleapis/mockgcp/cloud/notebooks/v1/*.proto \
```

Comment:
Should we also remove the generated files under `mockgcp/generated/mockgcp/cloud/notebooks/v1`?